### ISSUE TITLE
[Other - Accessibility] Unnecessary/overly verbose screen reader text. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#55864]
#### VFS team name
VA Online Scheduling (VAOS)

#### VFS product name
VA Online Scheduling

#### VFS feature name
Appointment List

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
There are a few instances of screen-reader-only text that can end up being distracting for screen reader users rather than providing necessary clarification or context.
- The first cell for each appointment is announced as "appointment on [date/time]." The "appointment on" text isn't necessary given the context of the page, and the repetition ends up being cumbersome and distracting. Just an `aria-label` with the date/time string as formatted for screen reader announcing is sufficient.
- The H2 month/year headings are announced as "Appointments in [month/year]." From the context of the page, just the month/year are sufficient, no need to add the extra `<span class="sr-only">Appointments in</span>`.
- The modality of each appointment has the word "appointment" at the end of the `aria-label`, eg. "Video appointment" or "At Cheyenne VA Medical Center appointment." The "appointment" reminder doesn't really add much value.
- Each list of appointments for a given month is associated with the H2 that precedes it using `aria-labelledby`. The relationship between the list and the H2 is clear enough just from the DOM order and the month/year information is included for each appointment via another `aria-label`, so the `aria-labelledby` leads to some redundant screen reader behavior.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
I appreciate the intention here! It's just a little overzealous, and the repetition of the word "appointment" doesn't really benefit screen reader users. Definitely add screen-reader-only text (either the `sr-only` class or an `aria-label`) to give an interactive element a unique accessible name or to provide missing context that's communicated to sighted users by visual cues. But in these cases I don't think the extra clarification is needed.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by shiragoodman** [comment]
thanks @ldelacosta . Per our guidance above, we request that this ticket remain open until the issue is resolved. Thanks for your understanding.

**Comment by outerpress** [comment]
@ldelacosta I found this one digging through old tickets. Background, we had had plans for more accessibility improvements that we were going to wrap these into, but our a11y experts were in flux and that effort got dropped. I added dev ticket to address these: #93285

**Comment by outerpress** [comment]
Dev ticket is complete so this is resolved, closing (fyi @ldelacosta @Bren22va @JRRoof)
